src/Raven.Studio/wwwroot/App/views/database/documents/editDocument.html

```diff
@@ -225,11 +223,11 @@ <h3 class="margin-none">Properties</h3>
                             <div class="badge" data-bind="text: $root.crudActionsProvider().countersCount() || ''"></div>
                         </button>
                     </div>
-                    <div class="btn-group" id="revisions_pane" role="group" data-bind="visible: revisionsEnabled">
-                        <button type="button" class="btn btn-xs btn-default doc-revisions" data-bind="css: { active: isRevisionsActive() }, click: _.partial(activateRevisions, false)">
+                    <div class="btn-group" id="revisions_pane" role="group">
+                        <button type="button" class="btn btn-xs btn-default doc-revisions" data-bind="css: { active: isRevisionsActive() }, click: _.partial(activateRevisions, false), disable: !$root.canViewRevisions()">
```

Comment:
do we want to hide or disable revisions tab? What's the logic for other tabs?

Reply:
1. Revisions tab is always _visible_, never _hidden_.
2. Revisions tab is _disabled_ upon 'new document' & 'clone'